src/backend/joanie/core/serializers/admin.py

```diff
@@ -636,6 +636,30 @@ class Meta:
         read_only_fields = ["id", "course", "course_runs", "is_graded", "position"]
 
 
+class AdminOrderGroupSerializer(serializers.ModelSerializer):
+    """
+    Admin Serializer for OrderGroup model
+    """
+
+    nb_available_seats = serializers.SerializerMethodField(read_only=True)
+
+    class Meta:
+        model = models.OrderGroup
+        fields = [
+            "id",
+            "nb_seats",
+            "product",
+            "is_active",
+            "nb_available_seats",
+            "created_on",
+        ]
+        read_only_fields = ["id", "created_on"]
```

Comment:
`product` is still missing

Reply:
oh so it's a problem, we should have different serializer depending on the request action because once created this should never change.
This is out the scope of this PR I guess and should be made in an other one.

Reply:
I create a dedicated serializer to address OrderGroup creation.